<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1215</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Packet Capture</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>20</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Palatino Linotype</family>
     <pointsize>16</pointsize>
     <italic>false</italic>
     <fontweight></fontweight>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(32, 54, 143);
font: 75 16pt &quot;Palatino Linotype&quot;;</string>
   </property>
   <property name="text">
    <string>Network Traffic Monitoring</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>460</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 35, 51);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Start Capture</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>460</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 35, 51);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Stop Capture</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>1011</width>
     <height>341</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Palatino Linotype</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <fontweight></fontweight>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 9pt &quot;Palatino Linotype&quot;;
QTableWidget {
    border: 2px solid #7084EA;
    border-radius: 10px;
    background-color: #F0F4FF;
    padding: 10px;
}
QTableWidget:hover {
    border: 2px solid #5064CA;
    background-color: #E8EBFF;
}</string>
   </property>
   <column>
    <property name="text">
     <string>Source IP</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Destination IP</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Source MAC</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Destination MAC</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Protocol</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>flags</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Info</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Sport</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Dport</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Timestamp</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>450</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 35, 51);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Download Packets</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 35, 51);</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QComboBox" name="packet_type">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <item>
    <property name="text">
     <string>TCP </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ARP</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ICMP </string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="flags">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>URG</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ACK</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PSH</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>RST</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SYN</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>FIN</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ECE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CWR</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>NS</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
